<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QWidget" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Vibe Snitch - AI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
       border-radius: 20px;
       background-color: #1a1328;
}</string>
  </property>
  <widget class="QFrame" name="mainCard">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#mainCard {
    background-color: #1a1328;
    border-radius: 20px;
    border: 2px solid #26183d;
}


</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>42</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #dbd2d4;
background-color: #1a1328;

</string>
      </property>
      <property name="text">
       <string>Vibe Snitch</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSlogan">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #dbd2d4;
background-color: #1a1328;

</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drop their posts, and weâ€™ll snitch their vibes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtName">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #120c20;
color:#C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 3px 15px;
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Their Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPost01">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #120c20;
color:#C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 15px;
</string>
      </property>
      <property name="placeholderText">
       <string>Share the thoughts here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPost02">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #120c20;
color: #C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 15px;
</string>
      </property>
      <property name="placeholderText">
       <string>Share the thoughts here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPost03">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #120c20;
color: #C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 15px;
</string>
      </property>
      <property name="placeholderText">
       <string>Share the thoughts here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPost04">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #120c20;
color: #C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 15px;
</string>
      </property>
      <property name="placeholderText">
       <string>Share the thoughts here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPost05">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #120c20;
color: #C8C8C8;
border: 1px solid #2D0B4E;
border-radius: 15px;
padding: 15px;
</string>
      </property>
      <property name="placeholderText">
       <string>Share the thoughts here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3a1a6e;
color: #dbd2d4;
border-radius: 15px;
padding: 12px;

}

QPushButton:hover {
    background-color: #892CDC;
}
</string>
      </property>
      <property name="text">
       <string>Predict Personality</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
